test/falco_tests.yaml

```diff
@@ -1169,7 +1169,7 @@ trace_files: !mux
   skip_unknown_error:
     exit_status: 1
     stderr_contains: |+
-      Could not load rules file.*skip_unknown_error.yaml: 1 errors:
+      Could not compile rules in the engine: 1 errors:
```

Comment:
I hear your point and I think you're right. I definitely think we should collect loading info/errors/warnings more accurately, but I agree this may be too much for this single PR.